gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt

```diff
@@ -1204,3 +1204,28 @@ drunkard = {
 		}
 	}
 }
+
+magic_good = {
```

Comment:
You should visit `wc_ethnicities_elven.txt` and remove the random gold eye color option from the night elves.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/119353618-2a9d0200-bcb4-11eb-8607-71172b51a1ef.png)

</details>

Reply:
I knew you mentioned this was in the original code, but I couldn't find it when I was looking around. Thank you for pointing that out! I will update this in my next commit.

Reply:
Shouldn't this also be deleted from the highborne?